framework/wazuh/core/config/models/tests/test_central_config.py

```diff
@@ -1,19 +1,19 @@
 import pytest
 
-from wazuh.core.config.models.central_config import Config, EngineConfig, ManagementAPIConfig, CommsAPIConfig
+from wazuh.core.config.models.central_config import Config, EngineConfig, ManagementAPIConfig, CommsAPIConfig, IndexerConfig
 
 
-@pytest.mark.parametrize("init_values, expected", [
+@pytest.mark.parametrize('init_values, expected', [
     ({
-        "indexer": {"host": "localhost", "port": 9200, "user": "user_example", "password": "password_example"},
-        "server": {"nodes": ["master"], "node": {"name": "example", "type": "master", "ssl":
-            {"key": "value", "cert": "value", "ca": "value"}}}
+        'indexer': {'hosts': [{'host': 'localhost', 'port': 9200}], 'username': 'user_example', 'password': 'password_example'},
+        'server': {'nodes': ['master'], 'node': {'name': 'example', 'type': 'master', 'ssl':
+            {'key': 'value', 'cert': 'value', 'ca': 'value'}}}
      },
      {
         "node": {"name": "example", "type": "master", "ssl": {"key": "value", "cert": "value", "ca": "value"}},
         "server": {"nodes": ["master"], "port": 1516, "bind_addr": "localhost", "hidden": False, "update_check": False,
-                   "logging.level": "debug2"},
-        "indexer": {"host": "localhost", "port": 9200, "user": "user_example", "password": "password_example"},
+                   "logging.level": "info"},
```

Comment:
Use single quotes

Reply:
It was changed in edcb389a839c82846356d2ca9e981f422df1c442.